packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/ExistingInvoiceDetails.tsx

```diff
@@ -221,6 +235,14 @@ export const ExistingInvoiceDetails = (
                     disabled={loading}
                   >{t(i18n)`Delete`}</Button>
                 )}
+                {buttons.isEditButtonVisible && (
+                  <Button
```

Comment:
Alternatively, you can add permissions checking right here. Btw, _Delete_ action is missing permission checking too. Check out [an example](https://github.com/team-monite/monite-sdk/commit/bde068d520a604abf069d910e48711a36fe58039#diff-4b85029f45ba03ebdfa44b46e4f56a24dfa7c864daac70d2aaeafc61b6f1faeaR72-R76)!

Reply:
We don't support permissions right now